console/account/how-to/report-incident.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: How to report an incident
+  description: This page shows you how to submit an abuse report and to indicate how the report will be processed.
+content:
+  h1: How to report an incident
+  paragraph: This page shows you how to submit an abuse report and to indicate how the report will be processed.
+tags: report incident
+dates:
+  validation: 2024-02-19
+  posted: 2024-02-19
+categories:
+  - account
+---
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+</Message>
+
+## How to report an incident
+1. From your [Scaleway console](https://console.scaleway.com) or from the Scaleway website click the Report Abuse page. A form displays.
+2. Enter the IP address, domain name or URL source of your report.
+3. Enter the publication date of the illegal content or the start of the illegal behavior.
+4. Enter the time of the publication of the illegal content or the start of the illegal behavior.
+5. Select the type of abuse you wish to report in the drop-down box
+    <Message type="tip">
+      The different types of content or illegal behavior that can be reported is described in Scaleway’s [Abuse notice](https://www.scaleway.com/en/abuse-notice/).
+    </Message>
+    <Message type="important">
+      We hereby remind you that reporting content or activity as illegal in order to obtain its removal or to stop its distribution, when you do know the information are indeed inaccurate, is punishable by one year imprisonment and a €15,000 fine.
+    </Message>
+6. Enter your email address and name if required to do so.
+7. Describe what you have observed in as much detail as possible. Provide information such as logs, time zones, headers, the precise location and if applicable, the email addresses to which the content is made accessible.
+    <Message type="important">
+      This description will be directly forwarded to our customer. Do not therefore include any confidential or personal information which you do not want them to have access to. 
+      Scaleway denies all responsibility for the use which could be made by third parties of any information that you provided in this field.
+    </Message>
+8. Check the box confirming that you are submitting this report in good faith.
+9. Once you have entered the information mentioned above, check the CAPTCHA form and click **Submit abuse report**.
+    <Message type="note">
+      In case you encounter difficulties to submit an abuse, you can contact [Scaleway’s abuse team](https://www.scaleway.com/fr/contact-trust-et-safety/).
```

Comment:
```suggestion
      In case you encounter difficulties to submit an abuse report, you can contact [Scaleway’s abuse team](https://www.scaleway.com/fr/contact-trust-et-safety/).
```